.github/workflows/ci-actions-incremental.yml

```diff
@@ -324,7 +327,7 @@ jobs:
       - name: Build
         shell: bash
         # Despite the pre-calculated run_jvm flag, GIB has to be re-run here to figure out the exact submodules to build.
-        run: ./mvnw $COMMON_MAVEN_ARGS clean install -Dsurefire.timeout=1200 -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devtools -pl !docs ${{ matrix.java.maven_args }} ${{ needs.build-jdk11.outputs.gib_args }}
+        run: ./mvnw $COMMON_MAVEN_ARGS clean install -Dsurefire.timeout=1200 -pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/devmode -pl !integration-tests/devtools -pl !:quarkus-integration-test-kubernetes-invoker -pl !:quarkus-integration-test-kubernetes-standard -pl !:quarkus-integration-test-kubernetes-standard-kafka -pl !docs ${{ matrix.java.maven_args }} ${{ needs.build-jdk11.outputs.gib_args }}
```

Comment:
Three very minor remarks/hints:
- in https://github.com/quarkusio/quarkus/blob/main/integration-tests/kubernetes/pom.xml we could move all modules to a profile that is always activated (there should be examples elsewhere) and deactivate it here via `-P !...` -> would probably make this a bit easier to read
- we now have a mixture of artifactIds (`-pl :...`) and paths -> unify?
- `-pl` can also take multiple modules -> would save a few chars

Reply:
Yes, good idea. I was not very happy about this either.

Reply:
> we now have a mixture of artifactIds (-pl :...) and paths -> unify?

I agree it's not ideal but I don't see any better solution for now.